demos/storybook/stories/selection-toolbar/with-full-config.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { text } from '@storybook/addon-knobs';
+
+export const withFullConfig = (): any => ({
+    styles: [
+        `
+       ::ng-deep .pxb-selection-toolbar {
+           width: 600px !important;
+       }
+    `,
+    ],
+    template: `
+       <pxb-selection-toolbar [title]="title" [subtitle]="subtitle">
+            <button mat-icon-button pxb-icon>
+                <mat-icon>menu</mat-icon>
+            </button>
+        <ng-container pxb-menu>
+            <button mat-menu-item>
+                <span>Test Item 1</span>
+            </button>
+            <button mat-menu-item>
+                <span>Test Item 2</span>
+            </button>
+            <button mat-menu-item>
+                <span>Test Item 3</span>
+            </button>
+        </ng-container>
+        <button mat-icon-button pxb-right-content>
+                <mat-icon>home</mat-icon>
+            </button>
+       </pxb-selection-toolbar>
+    `,
+    props: {
+        title: text('title', 'Title'),
+        subtitle: text('subtitle', 'Subtitle'),
```

Comment:
Can you add a slider in here to toggle more or less buttons to show on the right?

Reply:
Add a knob to show what it looks like without a subtitle/menu?